src/pulsar-binding.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+const binary = require('@mapbox/node-pre-gyp');
+
+const bindingPath = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
Would this work on Windows?

Reply:
I'm starting to package Windows and I'll test it